Marlin/src/gcode/motion/G2_G3.cpp

```diff
@@ -274,6 +280,19 @@ void GcodeSuite::G2_G3(const bool clockwise) {
 
     get_destination_from_command();
 
+    #if ENABLED(LASER_MOVE_POWER)
+      // Set the laser power in the planner to configure this move
+      if (parser.seen('S')) {
+        cutter.inline_power(
+          #if ENABLED(SPEED_POWER_FLOAT)
+            parser.value_float()
+          #else
+            parser.value_int()
+          #endif
+        );
+      }
+    #endif
```

Comment:
This can probably be localized in `get_destination_from_command`, then all motion commands (G0,G1,G2,G3,G5) will automatically get this behavior.

Reply:
Thanks; That's a good idea